storage: TestTxnPutOutOfOrder failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/ba49c4f0b28468a436e8c41873c94992d9e6b71b

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=true
TAGS=
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=217591&tab=buildLog

```
I170412 06:39:01.487523 33114 gossip/gossip.go:293  [n1] NodeDescriptor set to node_id:1 address:<network_field:"" address_field:"" > attrs:<> locality:<> 
I170412 06:39:01.491597 33031 storage/split_queue.go:92  [split,s1,r1/1:/M{in-ax},@c4200a6000] splitting at key /System/""
I170412 06:39:01.491824 33114 storage/store.go:1256  [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170412 06:39:01.492477 33031 storage/replica_command.go:2602  [split,s1,r1/1:/M{in-ax},@c4200a6000] initiating a split of this range at key /System/"" [r2]
I170412 06:39:01.495604 33031 storage/split_queue.go:92  [split,s1,r2/1:/{System/""-Max},@c42045a900] splitting at key /System/tsd
I170412 06:39:01.495635 33031 storage/replica_command.go:2602  [split,s1,r2/1:/{System/""-Max},@c42045a900] initiating a split of this range at key /System/tsd [r3]
I170412 06:39:01.497234 33114 storage/replica_command.go:175  [s1,r1/1:/{Min-System/""},@c4200a6000] test injecting error: Test
W170412 06:39:01.498085 33031 storage/stores.go:218  range not contained in one range: [/Meta2/System/tsd,/System/tsd/NULL), but have [/Min,/System/"")
I170412 06:39:01.498965 33114 util/stop/stopper.go:494  quiescing; tasks left:
2      kv/txn_coord_sender.go:903
1      storage/queue.go:523
I170412 06:39:01.498996 33114 util/stop/stopper.go:494  quiescing; tasks left:
1      storage/queue.go:523
W170412 06:39:01.499165 33175 storage/intent_resolver.go:266  failed to resolve intents: node unavailable; try another peer
panic: llrb: inverted range

goroutine 33175 [running]:
github.com/cockroachdb/cockroach/vendor/github.com/biogo/store/llrb.(*Tree).DoRange(0xc42005e758, 0xc421081a18, 0x29b4000, 0xc4202d4870, 0x29b4000, 0xc4202d48d0, 0x69a5ef)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/biogo/store/llrb/llrb.go:505 +0x118
github.com/cockroachdb/cockroach/pkg/util/cache.(*OrderedCache).DoRange(0xc42005e700, 0xc421081bb0, 0x1b96b20, 0xc42016b0c0, 0x1b96b20, 0xc42016b0e0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/cache/cache.go:428 +0xfd
github.com/cockroachdb/cockroach/pkg/kv.(*rangeDescriptorCache).clearOverlappingCachedRangeDescriptors(0xc420ec0b90, 0x7f4638804b90, 0xc420945440, 0xc42159b4a0, 0x0, 0x11)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_cache.go:609 +0x3e1
github.com/cockroachdb/cockroach/pkg/kv.(*rangeDescriptorCache).insertRangeDescriptorsLocked(0xc420ec0b90, 0x7f4638804b90, 0xc420945440, 0xc42159b4a0, 0x1, 0x1, 0x7f4638804b00, 0xc420083970)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_cache.go:547 +0x91
github.com/cockroachdb/cockroach/pkg/kv.(*rangeDescriptorCache).lookupRangeDescriptorInternal.func3(0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_cache.go:329 +0x31f
github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/singleflight.(*Group).doCall(0xc420ec0bc8, 0xc420a62460, 0xc420f45af0, 0x6, 0xc42103db80)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/singleflight/singleflight.go:93 +0x3c
created by github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/singleflight.(*Group).DoChan
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/singleflight/singleflight.go:86 +0x323

goroutine 1 [chan receive]:
testing.(*T).Run(0xc4201785b0, 0x1d42cbd, 0x14, 0x1daad38, 0xc42052fa01)
	/usr/local/go/src/testing/testing.go:698 +0x2f4
testing.runTests.func1(0xc4201785b0)
	/usr/local/go/src/testing/testing.go:882 +0x67
testing.tRunner(0xc4201785b0, 0xc42052fb78)
	/usr/local/go/src/testing/testing.go:657 +0x96
testing.runTests(0xc4200ef220, 0x29aada0, 0x19c, 0x19c, 0x6952c5)
	/usr/local/go/src/testing/testing.go:888 +0x2c1
testing.(*M).Run(0xc4201f7f20, 0xc42052fd70)
	/usr/local/go/src/testing/testing.go:822 +0xfc
github.com/cockroachdb/cockroach/pkg/storage_test.TestMain(0xc42052ff20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/main_test.go:59 +0x20a
main.main()
	github.com/cockroachdb/cockroach/pkg/storage/_test/_testmain.go:874 +0xf7

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x2dffa00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1020 +0x77
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:583 +0xd2

goroutine 6 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0x2dffa00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1043 +0x5b
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:584 +0xf6

goroutine 33034 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc421122080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33031 [chan receive]:
github.com/cockroachdb/cockroach/pkg/kv.(*rangeDescriptorCache).lookupRangeDescriptorInternal(0xc420ec0b90, 0x7f4638804b90, 0xc420945440, 0xc420f45523, 0x4, 0xd, 0xc42103da80, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_cache.go:345 +0x696
github.com/cockroachdb/cockroach/pkg/kv.(*rangeDescriptorCache).LookupRangeDescriptor(0xc420ec0b90, 0x7f4638804b90, 0xc420945440, 0xc420f45523, 0x4, 0xd, 0xc42103da80, 0x3fc3333333333300, 0xc42130a240, 0xc4209c4300, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_cache.go:243 +0x92
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).getDescriptor(0xc42000a800, 0x7f4638804b90, 0xc420945440, 0xc420f45523, 0x4, 0xd, 0xc42103da80, 0xc42130a200, 0x2faf080, 0x12a05f200, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:444 +0x8d
github.com/cockroachdb/cockroach/pkg/kv.(*RangeIterator).Seek(0xc421076048, 0x7f4638804b90, 0xc420945440, 0xc420f45523, 0x4, 0xd, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/range_iter.go:159 +0x1c9
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).divideAndSendBatchToRanges(0xc42000a800, 0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42004e9c0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:730 +0x574
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).Send(0xc42000a800, 0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42004e5b0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:630 +0x31f
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).Send(0xc42125c0b0, 0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42004e4e0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:426 +0x382
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).send(0xc4202cc7c0, 0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42004e4e0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:528 +0x1f2
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).send(0xc420438140, 0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:807 +0x307
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).(github.com/cockroachdb/cockroach/pkg/internal/client.send)-fm(0x7f4638804b90, 0xc420945440, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:413 +0x76
github.com/cockroachdb/cockroach/pkg/internal/client.sendAndFill(0x7f4638804b90, 0xc420945440, 0xc4210770f8, 0xc420507c00, 0x40, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:436 +0xf7
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Run(0xc420438140, 0x7f4638804b90, 0xc420945440, 0xc420507c00, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:413 +0xda
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).adminSplitWithDescriptor.func1(0x7f4638804b90, 0xc420945440, 0xc420438140, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_command.go:2672 +0x6db
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).Txn.func1(0x7f4638804b90, 0xc420945440, 0xc420438140, 0xc4204a8698, 0xc420021350, 0xc420021300)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:490 +0x43
github.com/cockroachdb/cockroach/pkg/internal/client.(*Txn).Exec(0xc420438140, 0x7f4638804b90, 0xc420945440, 0xc420010101, 0xc4204a86c0, 0xc4210775d0, 0xa66bd4)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/txn.go:612 +0xb6
github.com/cockroachdb/cockroach/pkg/internal/client.(*DB).Txn(0xc4202cc7c0, 0x7f4638804b90, 0xc420945440, 0xc4206957d0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/internal/client/db.go:491 +0x238
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).adminSplitWithDescriptor(0xc42045a900, 0x7f46387278c0, 0xc42103fe60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42013b230, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_command.go:2673 +0xa73
github.com/cockroachdb/cockroach/pkg/storage.(*splitQueue).process(0xc4204950f0, 0x7f46387278c0, 0xc42103fe60, 0xc42045a900, 0xc421220000, 0x13, 0x20, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/split_queue.go:99 +0x259
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processReplica(0xc42000b700, 0x7f4638804bd0, 0xc4206956b0, 0xc42045a900, 0xc420de4100, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:609 +0x3b8
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1.2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:514 +0xf2
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTask(0xc4200185a0, 0xc421077ea0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:240 +0x105
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:523 +0x368
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc421122020)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33135 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001f)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdfe0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33126 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000015)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdc40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33140 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1355 +0x518
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc42123d310)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33122 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000013)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cda20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33120 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000021)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cd9e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33037 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).waitAndProcess(0xc420278480, 0x7f46387ff0c8, 0xc420016e28, 0xed07fc285, 0xc41d438106, 0x2dfe4c0, 0xc420de4100, 0xc4200185a0, 0xc4200a6000, 0xc4200a60f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:207 +0x241
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1.1(0xc4200a6000, 0xc420393c80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:269 +0x90
github.com/cockroachdb/cockroach/pkg/storage.(*storeReplicaVisitor).Visit(0xc420393c80, 0xc420d960a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:350 +0x2ff
github.com/cockroachdb/cockroach/pkg/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scanner.go:271 +0x29f
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4211220e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33132 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001c)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdf00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33036 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4211220c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33118 [select]:
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).printStatsLoop(0xc42125c0b0, 0x7f46387ff0c8, 0xc420016e28)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:207 +0xb4a
github.com/cockroachdb/cockroach/pkg/kv.NewTxnCoordSender.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:191 +0x98
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc420494690)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33125 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000016)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdaa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33127 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000017)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdc60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33033 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc421122060)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33032 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc421122040)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33136 [semacquire]:
sync.runtime_Semacquire(0xc420356474)
	/usr/local/go/src/runtime/sema.go:47 +0x34
sync.(*WaitGroup).Wait(0xc420356468)
	/usr/local/go/src/sync/waitgroup.go:131 +0x7a
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc4203563f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:171 +0x31
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3439 +0x2a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc420495af0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33129 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001a)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cde80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33134 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001e)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdfc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33121 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000012)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cda00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33103 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*idAllocator).start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/id_alloc.go:133 +0x79c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc42032d220)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33138 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startCoalescedHeartbeatsLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3487 +0x167
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc420495b10)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33142 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1355 +0x518
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc42123d3b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33114 [semacquire]:
sync.runtime_notifyListWait(0xc420de4010, 0xc400000001)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4000)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Quiesce(0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:496 +0x1f2
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:439 +0x1f2
github.com/cockroachdb/cockroach/pkg/storage_test.TestTxnPutOutOfOrder(0xc420394000)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/client_replica_test.go:319 +0xc38
testing.tRunner(0xc420394000, 0x1daad38)
	/usr/local/go/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:697 +0x2ca

goroutine 33173 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).maybeAddToPurgatory.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:671 +0xb17
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4204aeea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33123 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000020)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cda40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33035 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4211220a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33139 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1226 +0x165
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc420bde000)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33030 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x39e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc421122000)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33131 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000018)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33137 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3456 +0x337
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc420495b00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33119 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:155 +0x55
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cd9c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33141 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1355 +0x518
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc42123d360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33115 [chan receive]:
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open.func2(0xc4202dcdc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:430 +0x44
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:431 +0x4c1

goroutine 33124 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000014)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cda80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33128 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc400000019)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cde60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33133 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001d)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdfa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66

goroutine 33130 [semacquire]:
sync.runtime_notifyListWait(0xc420de4790, 0xc40000001b)
	/usr/local/go/src/runtime/sema.go:298 +0x10b
sync.(*Cond).Wait(0xc420de4780)
	/usr/local/go/src/sync/cond.go:57 +0x89
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4203563f0, 0xc4200185a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xac
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x33
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4200185a0, 0xc4202cdea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x7d
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x66


ERROR: exit status 2

make: *** [stress] Error 1
73 runs completed, 1 failures, over 9m31s
Makefile:173: recipe for target 'stress' failed
```

--- a/pkg/storage/client_raft_test.go
+++ b/pkg/storage/client_raft_test.go
@@ -162,6 +162,10 @@ func TestStoreRecoverFromEngine(t *testing.T) {
 func TestStoreRecoverWithErrors(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	storeCfg := storage.TestStoreConfig(nil)
+	// Splits can cause our chosen keys to end up on ranges other than range 1,
+	// and trying to handle that complicates the test without providing any
+	// added benefit.
+	storeCfg.TestingKnobs.DisableSplitQueue = true
 	eng := engine.NewInMem(roachpb.Attributes{}, 1<<20)
 	defer eng.Close()
 
@@ -170,8 +174,8 @@ func TestStoreRecoverWithErrors(t *testing.T) {
 	func() {
 		stopper := stop.NewStopper()
 		defer stopper.Stop()
+		keyA := roachpb.Key("a")
 		storeCfg := storeCfg // copy
-		keyA := rg1Key("a")
 		storeCfg.TestingKnobs.TestingEvalFilter =
 			func(filterArgs storagebase.FilterArgs) *roachpb.Error {
 				_, ok := filterArgs.Req.(*roachpb.IncrementRequest)
@@ -207,7 +211,7 @@ func TestStoreRecoverWithErrors(t *testing.T) {
 	store := createTestStoreWithEngine(t, eng, false, storeCfg, stopper)
 
 	// Issue a no-op write to lazily initialize raft on the range.
-	keyB := rg1Key("b")
+	keyB := roachpb.Key("b")
 	incArgs := incrementArgs(keyB, 0)
 	if _, err := client.SendWrapped(context.Background(), rg1(store), incArgs); err != nil {
 		t.Fatal(err)

--- a/pkg/storage/client_replica_test.go
+++ b/pkg/storage/client_replica_test.go
@@ -183,7 +183,7 @@ func TestTxnPutOutOfOrder(t *testing.T) {
 	// key is selected to fall within the meta range in order for the later
 	// routing of requests to range 1 to work properly. Removing the routing
 	// of all requests to range 1 would allow us to make the key more normal.
-	key := rg1Key("key")
+	const key = "key"
 	// Set up a filter to so that the get operation at Step 3 will return an error.
 	var numGets int32
 
@@ -191,6 +191,10 @@ func TestTxnPutOutOfOrder(t *testing.T) {
 	defer stopper.Stop()
 	manual := hlc.NewManualClock(123)
 	cfg := storage.TestStoreConfig(hlc.NewClock(manual.UnixNano, time.Nanosecond))
+	// Splits can cause our chosen key to end up on a range other than range 1,
+	// and trying to handle that complicates the test without providing any
+	// added benefit.
+	cfg.TestingKnobs.DisableSplitQueue = true
 	cfg.TestingKnobs.TestingEvalFilter =
 		func(filterArgs storagebase.FilterArgs) *roachpb.Error {
 			if _, ok := filterArgs.Req.(*roachpb.GetRequest); ok &&

--- a/pkg/storage/client_test.go
+++ b/pkg/storage/client_test.go
@@ -25,7 +25,6 @@ client_*.go.
 package storage_test
 
 import (
-	"bytes"
 	"fmt"
 	"math/rand"
 	"net"
@@ -78,15 +77,6 @@ func rg1(s *storage.Store) client.Sender {
 		}
 		return ba
 	})
-}
-
-// rg1Key returns a key that will reliably be located within the first range.
-// This is a bit of a bandaid to ensure tests continue working as intended
-// until the rg1 method is removed.
-func rg1Key(key string) roachpb.Key {
-	// Ensure that the key will be located in the first range by giving it the
-	// meta2 prefix. The first range only contains meta keys as of April 2017.
-	return roachpb.Key(bytes.Join([][]byte{keys.Meta2Prefix, []byte(key)}, nil))
 }
 
 // createTestStore creates a test store using an in-memory

